rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/rm/ListNodeSourceCommand.java

```diff
@@ -48,9 +50,19 @@ public void execute(ApplicationContext currentContext) throws CLIException {
         if (statusCode(OK) == statusCode(response)) {
             RmStateView state = readValue(response, RmStateView.class, currentContext);
             NodeSourceView[] nodeSources = state.getNodeSource();
-            resultStack(currentContext).push(nodeSources);
+            List<NodeSourceView> filtered = new ArrayList<>(nodeSources.length);
+            for (NodeSourceView nodeSourceEvent : nodeSources) {
+                if (!"NODESOURCE_REMOVED".equalsIgnoreCase(nodeSourceEvent.getEventType())) {
+                    filtered.add(nodeSourceEvent);
+                } else {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00108)